support/events/message_test.go

```diff
@@ -86,8 +87,14 @@ func TestErrorMessages(t *testing.T) {
 			eventList := &corev1.EventList{
 				Items: test.events,
 			}
-			fakeObj := &corev1.Pod{}
-			client := fake.NewClientBuilder().WithLists(eventList).Build()
+			fakeObj := &corev1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					UID: fakeObjUID,
```

Comment:
I don't think this test ever tested anything - or, bugs in the old fake client meant that we passed in an object without a UID, sent a field selector for nothing, and got back every object ;)